readme/email_to_note.md

```diff
@@ -0,0 +1,27 @@
+# Email to note
+
+You can save your emails in Joplin Cloud by forwarding your emails to your Inbox Email address. The content of the email and its attachments will be saved in a note with the name of the email inside a notebook called Inbox. 
+
+All emails forwarded to your Inbox Email will use your Joplin Storage capacity.
```

Comment:
To make it clear, it's not some special storage capacity just for email-to-note: "All notes created this way will use, like any other note, your Joplin Cloud storage capacity."